server/backend/local-dev-window.env.bat

```diff
@@ -0,0 +1,16 @@
+SET POSTGRES_USER=***
+SET  POSTGRES_PASSWORD=****
```

Comment:
I wonder maybe some variable value in this file no need to be hidden? Could just copy from the `server/backend/local-dev.env`. And we might want to remove the extra white space at this line, thanks!!